pandas/core/indexes/api.py

```diff
@@ -1,6 +1,9 @@
-from pandas.core.indexes.base import (Index, _new_Index,  # noqa
-                                 _ensure_index, _get_na_value,
-                                 InvalidIndexError)
+from pandas.core.indexes.base import (Index,
+                                      _new_Index,
+                                      _ensure_index,
```

Comment:
this shouldn't need a noqa (if its line length, break it on the prarens

Reply:
I think some of these are unused  and just there to export as part of the API.

Reply:
k